modules/update-upgrading-cli.adoc

```diff
@@ -176,3 +176,22 @@ $ oc get clusterversion
 NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
 version   4.7.0       True        False         2m        Cluster version is 4.7.0
 ----
+
+. Now that the cluster is updated to the new version, confirm your nodes are upgraded:
```

Comment:
I don't think this needs to be part of every update.  For example, if you're just bumping along in your z stream, you probably don't need to proactively poll.  The only use case I can think of is when you are bumping a minor from 4.y to 4.(y+1) and want to ensure your compute is sufficiently modern before you deploy a workload that requires a new-in-4.(y+1) feature.  Is it worth making that distinction here?  Because in other cases, I expect that immediately post-core-update, some compute nodes will not have completed their update, and I don't think admins need to be alarmed when they see that.